### PR TITLE
Copy "informal" feature specifications to language repository.

### PR DESCRIPTION
**Pull request by lrhn**
(Includes file history).

### CODE REVIEW

File: archive/feature-specifications/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+of a feature specification is integrated into the language specification we
+expect the extra processing to give rise to additional clarifications and
+corrections, which means that a feature specification is expected to be 
+_nearly_ as complete and correct as the language specification.
+
+### Feature specifications in this directory
+
+This directory contains older feature specifications, newer ones can be
+found in, respectively should be submitted to, the language repository
+[here](https://github.com/dart-lang/language).
```
Comment (munificent) [inline archive/feature-specifications/README.md:27]: May want to adjust or remove this paragraph now that it lives in the language repo.

File: archive/feature-specifications/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The status of every feature specification in this directory is that it is
+background material, and the contents has been integrated into the language
```
Comment (munificent) [inline archive/feature-specifications/README.md:30]: "has" -> "have".

**Review by munificent — APPROVED** [review]
Instead of a separate "archive" directory, what do you think of moving them to "accepted/before 2.0" or something along those lines? That way, all the accepted feature specs are under "accepted", even the old ones.

File: archive/feature-specifications/README.md (hunk 1 above)
Comment (lrhn) [reply archive/feature-specifications/README.md]: Done.

File: archive/feature-specifications/README.md (hunk 2 above)
Comment (lrhn) [reply archive/feature-specifications/README.md]: Done.

**Comment by lrhn** [comment]
The CLA Override was used because these files are *already* published in a different repository, and this is just a move operation.

**Pull request merged**